pcsx2/CDVD/IsoHasher.cpp

```diff
@@ -106,6 +108,7 @@ void IsoHasher::Close()
 	if (!m_is_open)
 		return;
 
+	s_is_computing_hash = false;
```

Comment:
These atomic writes are in the wrong place. The variable should be set to true before the first CDVD call and set to false after the last CDVD call.

Reply:
fixed